buildsystem/compdeps/org.eclipse.persistence.compdeps/feature.xml

```diff
@@ -45,6 +45,111 @@
             version="@XJC_VERSION@"
             unpack="false"/>
 
+    <plugin
+            id="org.glassfish.corba.glassfish-corba-orb"
+            download-size="0"
+            install-size="0"
+            version="@GF_CORBA_ORB_VERSION@"
+            unpack="false"/>
+
+    <plugin
+            id="org.glassfish.corba.glassfish-corba-csiv2-idl"
+            download-size="0"
+            install-size="0"
+            version="@GF_CORBA_CSIV2_VERSION@"
+            unpack="false"/>
+
+    <plugin
+            id="org.glassfish.corba.glassfish-corba-internal-api"
+            download-size="0"
+            install-size="0"
+            version="@GF_CORBA_IAPI_VERSION@"
+            unpack="false"/>
+
+    <plugin
+            id="org.glassfish.corba.glassfish-corba-omgapi"
+            download-size="0"
+            install-size="0"
+            version="@GF_CORBA_OMGAPI_VERSION@"
+            unpack="false"/>
+
+    <plugin
+            id="org.glassfish.corba.exception-annotation-processor"
+            download-size="0"
+            install-size="0"
+            version="@GF_CORBA_EAP_VERSION@"
+            unpack="false"/>
+
+    <plugin
+            id="org.osgi.core"
+            download-size="0"
```

Comment:
Why import plugin (as in features/ope.jpa/feature.xml cannot be used here instead?